androidApp/src/main/res/values/strings.xml

```diff
@@ -118,6 +118,7 @@
     <string name="facility_issue">Facility Issue</string>
     <string name="facility_issue_sentence_case">Facility issue</string>
     <string name="favorites_link">Favorites</string>
+    <string name="no_stops_added" translatable="false">No stops added</string>
```

Comment:
I’d rather use `tools:ignore="MissingTranslation"` than `translatable="false"` so it’s clearer that there need to be translations eventually and that once they exist they should be used.